From 15dc7ba6e006d1c1c848ae6ab61b58942d835502 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 9 Apr 2011 12:12:42 +0800
Subject: [PATCH] Revert "modify some files according to A22 hardware"

This reverts commit ff59b4ed4f5022ca2a4ec2ccdf0d2f33d432c60b.

only revert
drivers/i2c/busses/i2c-rk29.c
drivers/mmc/host/rk29_sdmmc.c
ddr.c reverted by 4945717893f3fff2b7fc284fd606af376972aacd
---
 drivers/i2c/busses/i2c-rk29.c | 6 ------
 drivers/mmc/host/rk29_sdmmc.c | 4 ----
 2 files changed, 10 deletions(-)

diff --git a/drivers/i2c/busses/i2c-rk29.c b/drivers/i2c/busses/i2c-rk29.c
index b0733cecbcce..efc129f0e0fa 100755
--- a/drivers/i2c/busses/i2c-rk29.c
+++ b/drivers/i2c/busses/i2c-rk29.c
@@ -427,9 +427,6 @@ static int rk29_i2c_send_msg(struct rk29_i2c_data *i2c, struct i2c_msg *msg)
 		lsr = readl(i2c->regs + I2C_LSR);
 		if((lsr & I2C_LSR_RCV_NAK) && (i != msg->len -1) && !(msg->flags & I2C_M_IGNORE_NAK))
 			return -EINVAL;
-		
-		if(msg->addr == 0x41)	//delay for tp
-		udelay(50);
 
 	}
 	return ret;
@@ -458,9 +455,6 @@ static int rk29_i2c_recv_msg(struct rk29_i2c_data *i2c, struct i2c_msg *msg)
 		else
 			rk29_set_ack(i2c);
 		i2c_dbg(i2c->dev, "i2c recv >>>>>>>>>>>> buf[%d]: %x\n", i, msg->buf[i]);
-		
-		if(msg->addr == 0x41)	//delay for tp
-		udelay(50);
 	}
 	return ret;
 }
diff --git a/drivers/mmc/host/rk29_sdmmc.c b/drivers/mmc/host/rk29_sdmmc.c
index 80b070c11c15..7947a60cc34f 100755
--- a/drivers/mmc/host/rk29_sdmmc.c
+++ b/drivers/mmc/host/rk29_sdmmc.c
@@ -706,13 +706,9 @@ static int rk29_sdmmc_get_ro(struct mmc_host *mmc)
 
 static int rk29_sdmmc_get_cd(struct mmc_host *mmc)
 {
-#if defined(CONFIG_MACH_RK29_A22)	//A22's sdcard do not support detect pin
-return 1;
-#else
 	struct rk29_sdmmc *host = mmc_priv(mmc);
 	u32 cdetect = rk29_sdmmc_read(host->regs, SDMMC_CDETECT);
 	return (cdetect & SDMMC_CARD_DETECT_N)?0:1;
-#endif
 }
 
 static void rk29_sdmmc_enable_sdio_irq(struct mmc_host *mmc, int enable)
-- 
2.35.3

